examples/bri-3/src/bri/zkp/api/dtos/request/createProof.dto.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { plainToInstance } from 'class-transformer';
+import { validate } from 'class-validator';
+import { SHOULD_NOT_BE_EMPTY_VALIDATION_MESSAGE } from '../../../../shared/constants';
+import { CreateProofDto } from './createProof.dto';
+
+describe('CreateProofDto', () => {
+  it('should return error in case document not provided.', async () => {
+    // Arrange
+    const dto = {
+      ownerAccountId: '123',
+      signature: '123',
+    };
+    const createProofDto = plainToInstance(CreateProofDto, dto);
+
+    // Act
+    const errors = await validate(createProofDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('document');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'document ' + SHOULD_NOT_BE_EMPTY_VALIDATION_MESSAGE,
+    );
+  });
+
+  it('should return error in case signature not provided.', async () => {
+    // Arrange
+    const dto = {
+      document: {
+        documentObjectType: 'document',
+        documentObjectInput: { input: 'This is a document' },
+      },
+    };
+    const createProofDto = plainToInstance(CreateProofDto, dto);
+
+    // Act
+    const errors = await validate(createProofDto);
+
+    // Assert
+    expect(errors.length).toBe(2);
+    expect(errors[0].property).toEqual('signature');
+    expect(errors[0].constraints.isNotEmpty).toContain(
```

Comment:
Should we not also be checking for accountId error here as well? depending on your answer to mby above question.